src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger/config.jelly

```diff
@@ -236,12 +236,32 @@
            <f:checkbox name="silentMode"
                        checked="${it.silentMode}"/>
         </f:entry>
-        <f:entry title="${%Enable Topic Association}"
-                 field="enableTopicAssociation"
-                 help="/plugin/gerrit-trigger/trigger/help-EnableTopicAssociation.html">
-           <f:checkbox name="enableTopicAssociation"
-                       checked="${it.enableTopicAssociation}"/>
-        </f:entry>
+
+        <f:optionalBlock name="topicAssociation"
```

Comment:
I think this could be made a bit more sleek with a `config.jelly` view for the `TopicAssociation` class and a `<f:optionalProperty/>` here instead.